### PR TITLE
Notify all admins when questionnaire is submitted

### PR DESCRIPTION
**Pull request by roshan-mathewtech**
[36](https://unep-wcmc.codebasehq.com/projects/ors-maintainance/tickets/36)

Right now when a respondent submits the questionnaire notification is sent only to the admin user who created the questionnaire. This need to be changed so that all admins receive notification when submitted.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ gem 'jquery-ui-rails', '2.0.1'
 gem 'jqgrid-jquery-rails', '~> 4.4.001'
 gem 'tinymce-rails', '4.0'
 gem 'tinymce-rails-langs', '~>4.2'
-gem 'jstree-rails', :git => 'git://github.com/tristanm/jstree-rails.git'
+gem 'jstree-rails', :git => 'https://github.com/tristanm/jstree-rails.git'
```
Comment (lucacug) [inline Gemfile:16]: this github page recommend installing the gem from here now https://rails-assets.org/#/components/jstree
Have you tried that? Not sure what the issue with this gem was, but given we had to update the source maybe it is worth it to do it properly. Let's test if everything still works using the other source first

File: app/controllers/questionnaires_controller.rb [hunk 2]
```diff
@@ -287,7 +287,7 @@ def submit
       if @authorized_submitter.save!
         if Rails.env == 'production'
           UserMailer.questionnaire_submitted(current_user, @questionnaire).deliver
-          UserMailer.admin_notification_questionnaire_submitted(current_user, @questionnaire).deliver
```
Comment (lucacug) [inline app/controllers/questionnaires_controller.rb:290]: can we remove this method from the UserMailer file if we don't use it anymore, please?

**Review by lucacug — COMMENTED** [review]
just a couple of minor comments, looks good otherwise.

File: Gemfile (hunk 1 above)
Comment (roshan-mathewtech) [reply Gemfile:16]: I have not changed the source, just replaced the git:// protocol with https:// as git protocol was not fetching the gem for me, I think this is a known issue. 

https://stackoverflow.com/questions/21260689/force-bundle-install-to-use-https-instead-of-git-for-github-based-gems

Let me know your thought on this... thanks

File: app/controllers/questionnaires_controller.rb (hunk 2 above)
Comment (roshan-mathewtech) [reply app/controllers/questionnaires_controller.rb:290]: Done

**Comment by roshan-mathewtech** [comment]
Merging this PR as this need to go to live today

**Pull request merged**